### PR TITLE
Prepare release v0.1.14

### PR DESCRIPTION
**Pull request by nanzhong**
Updated changelog, and also updated the example manifest with the new flag

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
 # CHANGELOG
 
-## unreleased
+## v0.1.14 (beta) - Apr 26 2018
```
Comment (timoreimann) [inline CHANGELOG.md]:
```suggestion
## v0.1.14 (beta) - Apr 26th 2019
```

Comment (jcodybaker) [reply CHANGELOG.md]: Good catch on the year.  I assumed you were just tagging the `th`.

Comment (timoreimann) [reply CHANGELOG.md]: I was initially but luckily spotted it from the corner of my eye. :)

Comment (nanzhong) [reply CHANGELOG.md]: Welp, I'm stuck in the past.

**Pull request merged**